script.js

```diff
@@ -19,6 +20,171 @@ function closer() {
+                        <li><button class="btn">${popup[0].technologies[4]}</button></li>
+                    </ul>
+                    <button class="see pro">See Project</button>
+                </div>
+                <div class="grid-item">
+                    <h3 class="professional-btn">${popup[0].namepop}</h3>
+                    <p class="multiple-p">${popup[0].smalldescrip}</p>
+                    <ul class="languages">
+                        <li><button class="btn">${popup[0].technologies[1]}</button></li>
+                        <li><button class="btn boot">${popup[0].technologies[3]}</button></li>
+                        <li><button class="btn">${popup[0].technologies[4]}</button></li>
+                    </ul>
+                    <button class="see pro">See Project</button>
+                </div>
+                <div class="grid-item">
+                    <h3 class="professional-btn">${popup[0].namepop}</h3>
+                    <p class="multiple-p">${popup[0].smalldescrip}</p>
+                    <ul class="languages">
+                        <li><button class="btn">${popup[0].technologies[1]}</button></li>
+                        <li><button class="btn boot">${popup[0].technologies[3]}</button></li>
+                        <li><button class="btn">${popup[0].technologies[4]}</button></li>
+                    </ul>
+                    <button class="see pro">See Project</button>
+                </div>
+            </div>`;
+}
+
+let modalContainer;
+function popupWindow() {
+  modalContainer = document.createElement('section');
+  modalContainer.className = 'modalContainer';
+  modalContainer.innerHTML = `
+  <div class="popup-window">
+  <div class="multiflex">
+  <p class="x">&#x2716;</p>
+    <h2 class= "multih2">Multi Post Stories</h2>
+  </div>
+    <ul class="languages card1">
+        <li><button class="popbtn">${popup[0].technologies[1]}</button></li>
+        <li><button class="popbtn boot">${popup[0].technologies[3]}</button></li>
+        <li><button class="popbtn">${popup[0].technologies[4]}</button></li>
+    </ul>
+    <ul class="languages card4 cardplus">
+        <li><button class="popbtn">${popup[0].technologies[1]}</button></li>
+        <li><button class="popbtn boot">${popup[0].technologies[3]}</button></li>
+        <li><button class="popbtn ruby">${popup[0].technologies[2]}</button></li>
+    </ul>
+    <div class="multicontainer">
+    <img src="/Images/snapshot.png" alt="" class="snapshot-img">
+    <div class="descripcontainer">
+    <p>${popup[0].description}</p>
+    <button class="snapshot-btn1" onclick="javascript:window.open('${popup[0].live}', '_blank');">See Live <img src="/Images/live.png" class="btnpng"></button>
+    <button class="snapshot-btn" onclick="javascript:window.open('${popup[0].source}', '_blank');">See Source <img src="/Images/git.png" class="btnpng"></button>
+    </div>
+
+    </div>
+
+</div>`;
+  midpart.appendChild(modalContainer);
+}
+popupWindow();
```

Comment:
- Kindly make sure the images on your pop-up, see live and see source button respectively are rendering correcting 😉 as seen on the original Figma design 👍🏽
Your Implementation | Figma Design
:---:|:---:
![screenshot (15)](https://user-images.githubusercontent.com/44624138/156437893-75b9e37f-edd9-4723-8710-fbc85a9e666f.png)
| 
![screenshot (16)](https://user-images.githubusercontent.com/44624138/156437979-f065c94d-6e89-4dad-8739-c07f2e88a9a0.png)
![screenshot (17)](https://user-images.githubusercontent.com/44624138/156437983-dab9c6c8-0e89-48ce-bd6b-9fd77376fb8a.png)